Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,26 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            int age = Lesson3.Age(message);
+            return age;
         }
 
         public static string PromptString(string message)
         {
-            // To do: call your implementation. 
-            return "";
+            string wholename = Lesson3.WholeName(message); 
+            return wholename;
         }
 
         public static float PromptFloat(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            float heightAndWeight=Lesson3.HeightAndWeight(message);
```

Comment:
Please be consistent with your spacing :)

Reply:
Tried to remove inconsistent spacing.